spec/role_model_spec.rb

```diff
@@ -3,13 +3,15 @@
 describe RoleModel do
 
   let(:model_class) { Class.new }
+  let(:role_options) { {} }
 
   before(:each) do
+  options = role_options
     model_class.instance_eval do
       attr_accessor :roles_mask
       attr_accessor :custom_roles_mask
       include RoleModel
-      roles :foo, :bar, :third
+      roles :foo, :bar, :third, options
```

Comment:
why even assign `options` above? we could use `role_options` here directly, couldn't we?